### PR TITLE
feat(Item): add size & grade categories if applicable

### PR DESCRIPTION
**Pull request by ari-party**

### CODE REVIEW

File: Item/Item.lua [hunk 1]
```diff
@@ -410,10 +410,34 @@ function methodtable.setCategories( self )
...
+
 	--- Only set category if category_type value exists
 	if self.smwData[ translate( 'SMW_Type' ) ] ~= nil then
 		local typeCategory = translate( 'category_' .. string.lower( self.smwData[ translate( 'SMW_Type' ) ] ) )
-		if typeCategory ~= nil and typeCategory ~= 'category_' .. string.lower( self.smwData[ translate( 'SMW_Type' ) ] ) then table.insert( self.categories, typeCategory ) end
+		if typeCategory ~= nil and typeCategory ~= 'category_' .. string.lower( self.smwData[ translate( 'SMW_Type' ) ] ) then
+			table.insert( self.categories, typeCategory ) 
+			
+			if self.smwData[ translate( 'SMW_Size' ) ] ~= nil then
+				local sizeCategory = translate( 'SMW_Size' ) .. ' ' .. self.smwData[ translate( 'SMW_Size' ) ] .. ' ' .. lowercaseWords( translate( 'category_' .. string.lower( self.smwData[ translate( 'SMW_Type' ) ] ) ) )
```
Comment (octfx) [inline Item/Item.lua]: I think `lowercaseWords` is only applicable for categories on .tools.  
Where .tools uses for example `Size 1 generator`, .wiki would use `Größe 1 Generator`.

File: Item/Item.lua [hunk 2]
```diff
@@ -410,10 +410,34 @@ function methodtable.setCategories( self )
...
+				table.insert( self.categories, sizeCategory )
+			end
+		
+			if self.smwData[ translate( 'SMW_Grade' ) ] ~= nil then
+				local gradeCategory = translate( 'SMW_Grade' ) .. ' ' .. self.smwData[ translate( 'SMW_Grade' ) ] .. ' ' .. lowercaseWords( translate( 'category_' .. string.lower( self.smwData[ translate( 'SMW_Type' ) ] ) ) )
```
Comment (octfx) [inline Item/Item.lua]: See comment above :)

**Review by octfx — CHANGES_REQUESTED** [review]

File: Item/Item.lua (hunk 1 above)
Comment (ari-party) [reply Item/Item.lua]: does if locale (or lang) is `en`: use `lowercaseWords` sound good?

Comment (octfx) [reply Item/Item.lua]: Thats how I would do it. You should be able to get the current language by calling `lang:getCode()`.

**Review by ari-party — COMMENTED** [review]
No changes

**Review by octfx — APPROVED** [review]
Thanks!

**Pull request merged**